client_code/forms/routes/ballot/Read/form_template.yaml

```diff
@@ -2,7 +2,7 @@ container:
   type: ColumnPanel
   properties: {col_widths: '{}'}
 components:
-- type: Label
+    - type: Label
```

Comment:
sweep: this should not have changed and this is no longer valid yaml as a result.